Package.swift

```diff
@@ -36,15 +36,19 @@ let package = Package(
         .process("swiftlint.yml"),
       ]),
 
+    .testTarget(
+      name: "AirbnbSwiftFormatToolTests",
+      dependencies: ["AirbnbSwiftFormatTool"]),
+
     .binaryTarget(
-      name: "swiftformat",
-      url: "https://github.com/nicklockwood/SwiftFormat/releases/download/0.51.8/SwiftFormat.artifactbundle.zip",
-      checksum: "69dcca93d5f9e8104c35e5091a77b30551d1e01dd80473d1a8ac7b40196ad31d"),
+      name: "SwiftFormat",
+      url: "https://github.com/calda/SwiftFormat/releases/download/0.54-beta-5/SwiftFormat.artifactbundle.zip",
```

Comment:
Is there a non-beta version (e.g. 0.53)?

Reply:
they were on 0.53 here :

![image](https://github.com/slumberGroup/swift/assets/8989587/3232bf41-24f4-4d07-892b-6b08375cb30f)

https://github.com/airbnb/swift/pull/247/files

So even if we cherry picked into that commit (not sure how practical that would be) we would lose out on other commits, including the one that adds this [rule](https://github.com/airbnb/swift/pull/238) or this [one that bumps swiftlint](https://github.com/airbnb/swift/pull/239) which we could change/add outside but this would mean diverging further from airbnb.
Moreover, this version has almost 5 months and they haven't faced issues that deserved updating it.